Quadriga/src/main/java/edu/asu/spring/quadriga/service/transformation/impl/TransformationManager.java

```diff
@@ -51,4 +55,68 @@ public void saveTransformations(ITransformationFile transformationFile) throws F
         return transformationDAO.getAllTransformations();
     }
 
+    /**
+     * Retrieve the transformation file object which contains the absolute file
+     * paths of patternFile and MappingFile .
+     * 
+     * @param transformationId
+     *            Transformation ID of the transformation.
+     * @return Returns the transformation file object
+     */
+    @Transactional
+    @Override
+    public ITransformationFile retrieveTransformationFilePaths(String transformationId) {
```

Comment:
It should turn a transformationfiledto into a transformationfile and fill in the the transformation path.